Gaijinizer patch/Map398.txt

```diff
@@ -4,788 +4,992 @@
+When I looked behind me, there was a quiet woman with
+butterfly wings walking towards us.#Ron
 > END STRING
 
 > BEGIN STRING
 今のところ、殺気のような気配は感じられない……
 
 さっきの感じからするに、言葉も通じるようだ。
-> CONTEXT: Map398/events/1/pages/0/195/Dialogue < UNTRANSLATED
+> CONTEXT: Map398/events/1/pages/0/195/Dialogue
+I can't sense any murderous intent from her at this time.
 
+Based on her current demeanor, it looks like I'll be able
+to communicate with her.#RonIffy
 > END STRING
 
 > BEGIN STRING
 ……もしかして、あなたが『ティターニア』？
 
 この世界の、『妖精の女王』っていう………
-> CONTEXT: Map398/events/1/pages/0/199/Dialogue/kazuya < UNTRANSLATED
-
+> CONTEXT: Map398/events/1/pages/0/199/Dialogue/kazuya
+I don't suppose you're Titania, the queen of the
+Fairy Realm, are you?#Ron
 > END STRING
 
 > BEGIN STRING
 まさしく。
 
 私は誇り高きティターン族の末裔にして、
 全ての妖精を統べる女王…。
-> CONTEXT: Map398/events/1/pages/0/203/Dialogue/enemy28 < UNTRANSLATED
+> CONTEXT: Map398/events/1/pages/0/203/Dialogue/enemy28
+That I am.
 
+I'm a descendant of the proud race known as
+titans, and the ruler of all fairies.#Ron
 > END STRING
 
 > BEGIN STRING
 そして、あなたがた人間の親しき隣人であり
 
 その儚き命を憂いている者です。
-> CONTEXT: Map398/events/1/pages/0/208/Dialogue/enemy28 < UNTRANSLATED
-
+> CONTEXT: Map398/events/1/pages/0/208/Dialogue/enemy28
+I am also a friendly neighbor of you humans, and
+one who deeply mourns your fleeting lives.#RonIffyWording
 > END STRING
 
 > BEGIN STRING
 あなたの疑問は分かっています。\\N[1]。
 
 『私たちが親しき者と言うならば、
 　何故人間をそんな姿にするのか？』
-> CONTEXT: Map398/events/1/pages/0/215/Dialogue/enemy28 < UNTRANSLATED
-
+> CONTEXT: Map398/events/1/pages/0/215/Dialogue/enemy28
+I understand your misgivings, \\N[1].
+You wish to know how I can call myself a friendly
+neighbor when I'm changing humans into such a
+form.#RonIffyWording
 > END STRING
 
 > BEGIN STRING
 しかし、これは先に答えた通り……
 
 あなた方人間に対しての『慈悲』に他なりません。
-> CONTEXT: Map398/events/1/pages/0/220/Dialogue/enemy28 < UNTRANSLATED
+> CONTEXT: Map398/events/1/pages/0/220/Dialogue/enemy28
+But the reason for that is as I said...
 
+It's nothing more than "mercy" towards you
+humans.#Ron
 > END STRING
 
 > BEGIN STRING
 『慈悲』……？
-> CONTEXT: Map398/events/1/pages/0/224/Dialogue/kazuya < UNTRANSLATED
-
+> CONTEXT: Map398/events/1/pages/0/224/Dialogue/kazuya
+"Mercy"...?#Ron
 > END STRING
 
 > BEGIN STRING
 あなた方人間が、自由に生を謳歌できる時間……
 『寿命』は、あまりにも短い。
 
 我々は、それを古より嘆き、哀れんでおりました。
-> CONTEXT: Map398/events/1/pages/0/226/Dialogue/enemy28 < UNTRANSLATED
-
+> CONTEXT: Map398/events/1/pages/0/226/Dialogue/enemy28
+The time you humans have to freely enjoy your
+lives, your "lifespan", is very short. We fairies
+have pitied and lamented this fact since ancient
+times.#Ron
 > END STRING
 
 > BEGIN STRING
 そして我が王、オベロンとの話し合いの末……
 
 人間を少しずつですが、この妖精の国へと
 招き入れることにしたのです
-> CONTEXT: Map398/events/1/pages/0/231/Dialogue/enemy28 < UNTRANSLATED
-
+> CONTEXT: Map398/events/1/pages/0/231/Dialogue/enemy28
+So after discussing it with my king, Oberon, we
+decided to invite humans little by little to our
+Fairy Realm...#RonIffyWording
 > END STRING
```

Comment:
I'm not sure how much the initial text makes it clear, but my assumption that Oberon was her (ex)-husband, and not her king?